tests.py

```diff
@@ -73,6 +73,24 @@ def test_show_password(self):
         rv = self.app.get('/{0}'.format(key))
         self.assertTrue(password in rv.get_data(as_text=True))
 
+    def test_bots_denial(self):
+        """
+        Main known bots User-Agent should be denied access
+        """
+        password = "Bots can't access this"
+        key = snappass.set_password(password, 30)
+        a_few_sneaky_bots = [
```

Comment:
Should `facebot` also be included here? Perhaps this list should be build using variations of `SNEAKY_USER_AGENTS`.